src/api.ts

```diff
@@ -20,7 +21,7 @@ type Node = {
 export const createClient = async (
     connectTo?: string | Multiaddr | Node,
     peerIdOrSeed?: PeerId | string,
-): Promise<FluenceClient> => {
+): Promise<FluenceClientImpl> => {
```

Comment:
Exposing `FluenceClient` is intentional. We probably don't want to expose internals to public api